src/librustc_codegen_llvm/lib.rs

```diff
@@ -110,12 +110,13 @@ impl ExtraBackendMethods for LlvmCodegenBackend {
         ModuleLlvm::new_metadata(tcx, mod_name)
     }
 
-    fn write_metadata<'b, 'gcx>(
+    fn write_compressed_metadata<'b, 'gcx>(
```

Comment:
I think this name is misleading, it should probably be `{inject,embed,include,etc.}_compressed_metadata`.